lib/diagnostic.rb

```diff
@@ -93,13 +93,13 @@
 # In a Ruby string, write how you would remove the last two elements from this
 # array
 
-Diagnostic.remove_from_array = 'your answer here'
+Diagnostic.remove_from_array = 'arr.slice!(2..3)'
```

Comment:
The easiest way to do this would be `arr.pop 2` that way we don't need to know the length of the array.